cli.js

```diff
@@ -61,17 +63,32 @@ const run = (input, opts) => {
 	opts = Object.assign({plugin: DEFAULT_PLUGINS}, opts);
 
 	const use = requirePlugins(arrify(opts.plugin));
+	const spinner = ora('Minifying images');
 
 	if (Buffer.isBuffer(input)) {
 		imagemin.buffer(input, {use}).then(buf => process.stdout.write(buf));
 		return;
 	}
 
-	imagemin(input, opts.outDir, {use}).then(files => {
-		if (!opts.outDir) {
-			files.forEach(x => process.stdout.write(x.data));
-		}
-	});
+	if (opts.outDir) {
+		spinner.start();
+	}
+
+	imagemin(input, opts.outDir, {use})
+		.then(files => {
+			if (!opts.outDir) {
+				files.forEach(x => process.stdout.write(x.data));
```

Comment:
Yeah, this is a somewhat naive implementation expecting users that doesn't specify `--out-dir` only optimizing one file. I agree we should add an error for it since it won't work.